slider/src/main/java/io/monstarlab/mosaic/slider/Modifiers.kt

```diff
@@ -0,0 +1,74 @@
+package io.monstarlab.mosaic.slider
+
+import androidx.compose.foundation.gestures.Orientation
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.foundation.gestures.draggable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.progressSemantics
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.semantics.disabled
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.setProgress
+
+internal fun Modifier.sliderDragModifier(
+    state: SliderState,
+    enabled: Boolean,
+    interactionSource: MutableInteractionSource,
+    isRtl: Boolean,
+): Modifier = this.draggable(
+    state = state,
+    orientation = Orientation.Horizontal,
+    enabled = enabled,
+    interactionSource = interactionSource,
+    startDragImmediately = state.isDragging,
+    reverseDirection = isRtl,
+    onDragStopped = {},
```

Comment:
"onDragStopped" feels redundant here. I guess it could be removed.

Reply:
Removed :)